backend/src/main/java/com/eyetrade/backend/repository/CurrencyRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.eyetrade.backend.repository;
+
+import com.eyetrade.backend.model.entity.CurrencyRecord;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CurrencyRepository extends JpaRepository<CurrencyRecord,String> {
+    CurrencyRecord findTopByOrderByTimestamp();
```

Comment:
Maybe the last currency record would be cached in the memory to do not make a db call at each currency conversion request